auacm/app/modules/competition_manager/views.py

```diff
@@ -1,7 +1,6 @@
 from flask import request
 from flask.ext.login import current_user, login_required
 from app import app
-from app.database import session
```

Comment:
Cool. That makes sense. Just making sure that there was some sort of motivation.